hazelcast/src/test/java/com/hazelcast/internal/ascii/HTTPCommunicator.java

```diff
@@ -74,8 +74,13 @@ public String getClusterInfo() throws IOException {
     }
 
     public String getClusterHealth() throws IOException {
+        String suffix ="";
+        return getClusterHealthWithSuffix(suffix);
+    }
+
+    public String getClusterHealthWithSuffix(String suffix) throws IOException {
```

Comment:
Minor comment, can't help but thinking that this is a valid API, since I now have available a call which accepts suffix, but if I provide anything but empty string it will fail. So, I would suggest for testing purposes just hit directly the `doGet` rather than having this extra method.